### ISSUE TITLE
Queri::Realtime::Calls returns mismatched keys to values

### ISSUE THREAD

**Issue by collettiquette** [CTA/queri#1]
Set up:

``` ruby
Queri.configure_with(config) # config is path to yaml
Queri::Realtime::Calls.new([5000,10000])
```

Response: 

``` ruby
[{:direction_of_call=>"", :queue=>"IHS", :clid=>"+15077362004", :time_call_entered_queue=>"09:01:01", :time_call_on_wait=>"0:00", :call_duration=>"0:02", :agent=>"7:15", :music_on_hold=>"mmello", :server=>"", :magic_wand=>"&nbsp;"}, {:direction_of_call=>"", :queue=>"IHS", :clid=>"4062524294", :time_call_entered_queue=>"09:03:05", :time_call_on_wait=>"0:00", :call_duration=>"0:02", :agent=>"5:11", :music_on_hold=>"dwinslow", :server=>"", :magic_wand=>"&nbsp;"}, {:direction_of_call=>"", :queue=>"IHS", :clid=>"5076763100", :time_call_entered_queue=>"09:03:40", :time_call_on_wait=>"0:00", :call_duration=>"1:24", :agent=>"3:14", :music_on_hold=>"jhighfill", :server=>"", :magic_wand=>"&nbsp;"}, {:direction_of_call=>"", :queue=>"IHS", :clid=>"7737797824", :time_call_entered_queue=>"09:04:04", :time_call_on_wait=>"0:00", :call_duration=>"1:14", :agent=>"3:00", :music_on_hold=>"jinman", :server=>"", :magic_wand=>"&nbsp;"}, {:direction_of_call=>"", :queue=>"IHS", :clid=>"3367065475", :time_call_entered_queue=>"09:04:58", :time_call_on_wait=>"0:00", :call_duration=>"0:35", :agent=>"2:45", :music_on_hold=>"qquick", :server=>"", :magic_wand=>"&nbsp;"}, {:direction_of_call=>"", :queue=>"IHS", :clid=>"4238863019", :time_call_entered_queue=>"09:05:10", :time_call_on_wait=>"0:00", :call_duration=>"0:26", :agent=>"2:42", :music_on_hold=>"ajoy", :server=>"", :magic_wand=>"&nbsp;"}, {:direction_of_call=>"", :queue=>"IHS", :clid=>"6053431259", :time_call_entered_queue=>"09:05:26", :time_call_on_wait=>"0:00", :call_duration=>"0:13", :agent=>"2:39", :music_on_hold=>"pmccrimmons", :server=>"", :magic_wand=>"&nbsp;"}, {:direction_of_call=>"", :queue=>"IHS", :clid=>"2568357461", :time_call_entered_queue=>"09:05:30", :time_call_on_wait=>"0:00", :call_duration=>"0:46", :agent=>"2:02", :music_on_hold=>"sshipley", :server=>"", :magic_wand=>"&nbsp;"}, {:direction_of_call=>"", :queue=>"IHS", :clid=>"7703313171", :time_call_entered_queue=>"09:05:40", :time_call_on_wait=>"0:00", :call_duration=>"0:50", :agent=>"1:48", :music_on_hold=>"wwilson", :server=>"", :magic_wand=>"&nbsp;"}, {:direction_of_call=>"", :queue=>"IHS", :clid=>"4058630065", :time_call_entered_queue=>"09:06:01", :time_call_on_wait=>"0:00", :call_duration=>"0:55", :agent=>"1:22", :music_on_hold=>"sbarton", :server=>"", :magic_wand=>"&nbsp;"}, {:direction_of_call=>"", :queue=>"IHS", :clid=>"6039180483", :time_call_entered_queue=>"09:06:30", :time_call_on_wait=>"0:00", :call_duration=>"0:30", :agent=>"1:18", :music_on_hold=>"jblacksher", :server=>"", :magic_wand=>"&nbsp;"}, {:direction_of_call=>"", :queue=>"IHS", :clid=>"2153231230", :time_call_entered_queue=>"09:07:40", :time_call_on_wait=>"0:00", :call_duration=>"0:03", :agent=>"0:35", :music_on_hold=>"ahinton", :server=>"", :magic_wand=>"&nbsp;"}, {:direction_of_call=>"", :queue=>"IHS", :clid=>"2702312085", :time_call_entered_queue=>"09:08:11", :time_call_on_wait=>"0:00", :call_duration=>"0:02", :agent=>"0:05", :music_on_hold=>"lharness", :server=>"", :magic_wand=>"&nbsp;"}]
```

**Comment by collettiquette** [comment]
It seems to be just a few of the key value pairs that are mismatched.

**Comment by thermaldonkey** [comment]
This is fixed in commit 8b843bff9d322fb53566134a2a054e1bb596f9ce. Missing key in class declaration.
